### PR TITLE
Setup and Mobile Version Skeleton

### PR DESCRIPTION
**Pull request by shahedakhatoon**
## First Milestone in creating a Portfolio Website
1. Added a header section
2. Added a hero section
3. Added social media icons
4. Added a background image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        <a class="logo">My Logo</a>
+
+        <nav class="menu-icon">
+            <img src="src/Icon-Menu.png" alt="menu"/>
+        </nav>
+    </header>
+    <section class="headline">
+
+        <h2 class="title">I’m Mariane Glad to see you!
+            </h2>
```
Comment (BenMukebo) [inline index.html]: - [x] Please I would suggest you always use `h1` tag for your main website title, as for this, you can change this h2 to an h1 instead

File: index.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+                <li><a href=""><img class="icon msign" 
+                    src="src/m.png" alt="msign"/></a></li>
+
+                <li><a href=""><img class="icon github" 
+                    src="src/Github.png" alt="github-icon"/></a></li>
+
+                <li><a href=""><img class="icon finger" 
+                    src="src/Angel.png" alt="Angel-List"/></a></li>
+        </ul>
```
Comment (BenMukebo) [inline index.html]: - [x] Peeeeerfect 😄 ! you did a great job on wrapping those in a ul tag, however, this alignment is yet well arranged, see this image below and fix all as required, please.
- [x] One more thing is that, considering the Figma design, kindly adjust your margins, paragraph color, font-size and padding to fit the exact template, I will drop and image soon 😄 
- [x] You background image is not covering the section as it should, please adjust your properties to fit the original design.
h12

### consider making those changes to make design look like so ⬇️ 

hd

File: README.md [hunk 3]
```diff
@@ -1 +1,40 @@
-# Portfolio
\ No newline at end of file
+# Portfolio
+This is a mobile version skeleton of Portfolio.
+![Screenshot from 2022-02-16 17-06-35](https://user-images.githubusercontent.com/73285210/154257149-471f4cb2-fa17-4146-8ec8-397cc7cb1dd3.png)
+
+
```
Comment (BenMukebo) [inline README.md]: - [x] Great job, you have added an image to your readme, awesome it is so professional, but I think the image you have chosen is not the appropriate one, I will drop an image down to show you how you can take a screenshot of your application a clean way, take another picture and change update your readme, please.
tof

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### *** Hi 👋🏻  @shahedakhatoon ***, 


You've done well implementing some of the required tasks, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏 💻  

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Amrhub — APPROVED** [review]
# Approved 💯 ✅ 🥳 🎈 🎉 

Hi @shahedakhatoon, 👋🏻 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights ✅ 
Highly organized and clean code ✔️ 👨🏻‍💻 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**